legofy/__init__.py

```diff
@@ -77,22 +77,11 @@ def is_animated(im):
 
 
 def main(filename, brick, width=30, height=30, scale=1):
-    divider = "\\"
-    if "\\" not in filename:
-        divider = "/"
-    
     # open gif to start splitting
     baseImage = Image.open(filename)
     newSize = baseImage.size
     static = filename.lower().endswith(".gif") and is_animated(baseImage)
-    
-    fileDir = filename.split(divider)
-
-    newFilename = "lego_{0}".format(fileDir[len(fileDir) - 1])
-
-    del fileDir[len(fileDir) - 1]
-    fileDir = divider.join(fileDir)
-    
+    newFilename = '{0}/lego_{1}'.format(*os.path.split(filename))
```

Comment:
Please note that this will break Windows support. You can use `os.path.sep` instead of hardcoding the `/`, or use `os.path.join` to build the path.